src/ScriptBuilder/Saga/SagaScriptBuilder.cs

```diff
@@ -65,6 +65,9 @@ public static void BuildCreateScript(SagaDefinition saga, BuildSqlVariant sqlVar
 
             WriteComment(writer, "PurgeObsoleteProperties");
             sqlVariantWriter.WritePurgeObsoleteProperties();
+
+            WriteComment(writer, "CreateComplete");
```

Comment:
I think something like "Complete saga script" will be clearer when reading, given that the other two don't have it.

BTW we could review what common pieces we really have at some point, it gets a bit complicated and messy (e.g. Initialize() is empty for MSSQL, WriteTableNameVariable() empty for Oracle, etc.).